src/crypto/CHIPCryptoPALPSA.cpp

```diff
@@ -698,22 +712,61 @@ CHIP_ERROR P256Keypair::Initialize(ECPKeyTarget key_target)
     return error;
 }
 
+CHIP_ERROR P256Keypair::ImportRawKeypair(const uint8_t * private_key, const size_t private_key_size, const uint8_t * public_key,
+                                         const size_t public_key_size)
+{
+    psa_key_attributes_t attributes = configure_ecc_key_pair_attributes();
+    psa_key_id_t key_id;
+    psa_status_t status;
+
+    status = psa_import_key(&attributes, private_key, private_key_size, &key_id);
+    VerifyOrReturnError(status == PSA_SUCCESS, CHIP_ERROR_INTERNAL);
+
+    memcpy(mPublicKey.Bytes(), public_key, public_key_size);
+    memcpy(mKeypair.mBytes, &key_id, sizeof(psa_key_id_t));
+    mInitialized = true;
+
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR P256Keypair::ImportRawKeypair(const uint8_t * key_data, size_t key_data_size)
+{
+    CHIP_ERROR error                = CHIP_NO_ERROR;
+    psa_key_attributes_t attributes = configure_ecc_key_pair_attributes();
+    PsaP256KeypairContext & context = ToPsaContext(mKeypair);
+    psa_status_t status;
+
+    VerifyOrExit(key_data_size == kP256_PublicKey_Length + kP256_PrivateKey_Length, error = CHIP_ERROR_INTERNAL);
+
+    status = psa_import_key(&attributes, key_data + kP256_PublicKey_Length, kP256_PrivateKey_Length, &context.key_id);
+    VerifyOrExit(status == PSA_SUCCESS, error = CHIP_ERROR_INTERNAL);
+
+    memcpy(mPublicKey.Bytes(), key_data, kP256_PublicKey_Length);
+    memcpy(mKeypair.mBytes, &context.key_id, sizeof(psa_key_id_t));
+    mInitialized = true;
+
+exit:
+    logPsaError(status);
+
+    return error;
+}
+
 CHIP_ERROR P256Keypair::Serialize(P256SerializedKeypair & output) const
 {
-    CHIP_ERROR error                      = CHIP_NO_ERROR;
-    psa_status_t status                   = PSA_SUCCESS;
-    const PsaP256KeypairContext & context = ToConstPsaContext(mKeypair);
-    const size_t outputSize               = output.Length() == 0 ? output.Capacity() : output.Length();
+    CHIP_ERROR error        = CHIP_NO_ERROR;
```

Comment:
I kept the original implementation just to be able to use `PersistentStorageOperationalKeystore` with PSA, but if you don't want to support exporting the keys, I think it's better to just return an error when someone tries to call this function instead of serializing the keys in different formats than other platforms. Or is there a use case for this implementation?